### DISCUSSION TITLE
Example code for modbus RTU server WRITE_MULT_REGISTERS

### DISCUSSION THREAD

**Discussion by dialmak** [eModbus/eModbus#371] [Q&A]
Hi,
I am not a programmer. 
But I was able to understand the examples for modbus RTU server READ_HOLD_REGISTER, READ_INPUT_REGISTER and WRITE_HOLD_REGISTER.
Is it possible to see an simple example of processing WRITE_MULT_REGISTERS request for modbus RTU server?
Thanks.

**Comment by Miq1** [comment]
I will try to provide one later. Basically it will be not much different from the WRITE_REGISTER case: in the worker function, just use ``get()`` to extract start register address and number of registers from the request, then in a loop ``get()`` again to pull the values and store them at the requested location one by one.
